soc/arm/st_stm32/stm32f1/dts_fixup.h

```diff
@@ -2,6 +2,6 @@
 
 /* SoC level DTS fixup file */
 
-#define DT_ADC_1_NAME			DT_ST_STM32_ADC_40012400_LABEL
+#define DT_ADC_0_NAME			DT_LABEL(DT_INST(0, st_stm32_adc))
```

Comment:
@erwango is this really correct? I seem to recall there are more than one ADC on some STM32F1 parts, e.g. https://www.st.com/resource/en/datasheet/stm32f103rb.pdf.

I am asking because instance numbers are not soc numbers, so this can only be correct if there is just one. I do see that there's only one ADC node in stm32f1.dtsi, but should this beu sing DT_LABEL(DT_NODELABEL(adc1)) instead?

Sorry for my late review.